experimental/dds/tree-graphql/package.json

```diff
@@ -59,7 +59,7 @@
     "eslint": "~8.6.0",
     "mocha": "^10.0.0",
     "nyc": "^15.0.0",
-    "prettier": "~2.3.1",
+    "prettier": "~2.6.2",
```

Comment:
Didn't the tree packages' changes get merged in another PR?

Reply:
This change was made earlier than the Tree Prettier PR https://github.com/microsoft/FluidFramework/pull/13005.

I separated the PR later on so that it would be easier to apply the changes just for the Tree directories.

Reply:
Might be worth merging main into your branch / rebasing so the PR doesn't show the diff. But it's not vital.